### ISSUE TITLE
Unexpected type on the stack.

### ISSUE THREAD

**Issue by SimonCropp** [distantcam/ConfigureAwait#1]
[ConfigureAwait(false)]
        async void WithReaderAndWriter(TextWriter writer, StreamReader reader)
        {
            string line;
            while ((line = await reader.ReadLineAsync()) != null)
            {
                await writer.WriteLineAsync(line);
            }
        }

results in 
```
Error	44	Fody: PEVerify of the assembly failed.

Microsoft (R) .NET Framework PE Verifier.  Version  4.0.30319.0
Copyright (c) Microsoft Corporation.  All rights reserved.

[IL]: Error: [C:\Code\CaptureSnippets\CaptureSnippets\bin\Debug\CaptureSnippets.dll : 
CaptureSnippets.MarkdownProcessor+<WithReaderAndWriter>d__4::MoveNext][offset 0x0000004F]
[found ref 'System.Threading.Tasks.Task'][expected ref 'System.Threading.Tasks.Task`1[System.String]'] Unexpected type on the stack.(Error: 0x80131854)
[IL]: Error: [C:\Code\CaptureSnippets\CaptureSnippets\bin\Debug\CaptureSnippets.dll : 
CaptureSnippets.MarkdownProcessor+<WithReaderAndWriter>d__4::MoveNext][offset 0x00000058]
[found address of value 'System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1[System.String]'][expected address of value 'System.Runtime.CompilerServices.ConfiguredTaskAwaitable'] Unexpected type on the stack.(Error: 0x80131854)
2 Error(s) Verifying C:\Code\CaptureSnippets\CaptureSnippets\bin\Debug\CaptureSnippets.dll
	CaptureSnippets

```

**Comment by SimonCropp** [comment]
and it is only that method. no others in the project

**Comment by distantcam** [comment]
Will be fixed in 0.2
